devtools/quick_setup_script.sh

```diff
@@ -129,11 +129,11 @@ echo ~~~Setting up Contracts
 cd ../
 python -u devtools/contract_setup_script.py
 
-if [[ "mockData" != 'none' ]]; then
-    echo ~~~Creating mock data
-    cd ./app/server/utils
-    python -u mock_data.py ${testdata}
-fi
+#if [[ "mockData" != 'none' ]]; then
+#    echo ~~~Creating mock data
+#    cd ./app/server/utils
+#    python -u mock_data.py ${testdata}
```

Comment:
Are these commented out for testing, or are they broken with this PR?

Reply:
oh they're just broken in general, but this was me forgetting to uncomment